site/_includes/partials/toc-under-tree.njk

```diff
@@ -0,0 +1,11 @@
+{#
+  This is used as the TOC on smaller screens (<= 992) that makes use of the white space
+  under the navigation tree when the user scrolls down, in response to:
+  https://github.com/GoogleChrome/developer.chrome.com/issues/3097
+#}
+<div class="toc-container toc-container-under-navigation-tree display-none md:display-block xl:display-none pad-right-400 {% if not tocContents %}visibility-hidden{% endif %}">
```

Comment:
Why do you need this template? is it used somewhere?

Reply:
This is not needed and can be removed